  frame: /map
  #for segway
  #image_topic_name: /segbot_cam
  
  #for hexarotor
  image_topic_name: /hexa_cam
  
  sensor_field_of_view: 0.523
  sensor_distance: 6.0
  sensor_angular_res: 0.05
